elasticdl/Makefile

```diff
@@ -1,21 +1,22 @@
 PYTHON = python
-
-EDL_PROTO_FILE = elasticdl/proto/elasticdl.proto
-
+PROTO_PATH=elasticdl/proto
```

Comment:
I prefer we don't define this variable, as its naming doesn't add much information. Instead, we could literally use `elasticdl/proto` in the following rules.

Reply:
Done